util.go

```diff
@@ -173,6 +177,17 @@ func readBans() {
 	if err != nil && !os.IsNotExist(err) { // if there is an error and it is not a "file does not exist" error
 		Log.Println(err)
 		return
+	} else if err != nil {
```

Comment:
We don't need to create the file. We should just modify the previous statement so that it doesn't print a file does not exist error but does make the function return silently. The file will be created when writing bans.

Reply:
As the bans file should be written when using the ban command, it makes sense not to need it. Fair point.